### PR TITLE
Add icons for historic features

### PR DESCRIPTION
**Pull request by Yahia1018**
I am not sure if https://wiki.openstreetmap.org/wiki/File:Milk_churn.svg is displayed because I couldn't find one in OpenStreetMap and I have a doubt because the names don't match.

Capture d’écran 2022-01-11 à 11 14 57

What do we do in these cases ? Do we wait to encounter one in the map when working on a zone ?

EDIT : I found this incredibly useful page : https://taginfo.openstreetmap.org/, but apparently there is no such creamery poi in Switzerland. (EDIT : it doesnt't work for many keys finally...)

### CODE REVIEW

**Comment by bchapuis** [comment]
There are only 15 points with this attribute in the whole dataset. You can ignore it. In general, I think you can reasonably ignore the broken images.

**Comment by Yahia1018** [comment]
Ok thank you !

File: style.json [hunk 1]
```diff
@@ -861,8 +861,8 @@
...
...c_bookcase", "theatre", "courthouse", "firestation", "police", "post_box", "post_office", "prison", "townhall", "bbq", "bench", "drinking_water", "shelter", "shower", "telephone", "toilets", "recycling", "waste_basket", "waste_disposal", "internet_cafe", "public_bath" ] ] ], "rgba(115, 74, 10, 1)", [ "in", [ "get", "amenity" ], [ "literal", [ "kick-scooter_rental", "bicycle_parking", "bicycle_rental", "boat_rental", "bus_station", "car_rental", "charging_station", "fuel", "motorcycle_parking", "parking", "parking_entrance", "taxi" ] ] ], "rgba(0, 146, 218, 1)", [ "in", [ "get", "amenity" ], [ "literal", [ "clinic", "dentist", "doctors", "hospital", "pharmacy", "veterinary" ] ] ], "rgba(191, 0, 0, 1)", [ "in", [ "get", "amenity" ], [ "literal", [ "fountain" ] ] ], "rgba(87, 104, 236, 1)", [ "in", [ "get", "amenity" ], [ "literal", [ "marketplace" ] ] ], "rgba(172, 57, 172, 1)", [ "in", [ "get", "historic" ], [ "literal", [ "archaeological_site", "caslte", "creamery", "fort", "manor", "memorial", "monument" ] ] ], "rgba(115, 74, 10, 1)", [ "in", [ "get", "historic" ], [ "literal", [ "city_gate", "wayside_cross", "wayside_shrine" ] ] ], "rgba(85, 85, 85, 1)", "rgba(199, 116, 0, 1)" ],
```
Comment (bchapuis) [inline style.json:864]: Some amenity rules can be merged.

**Comment by Yahia1018** [comment]
I made an error sorry.

**Review by bchapuis — APPROVED** [review]

**Pull request merged**